source/includes/rankings.md

```diff
@@ -216,3 +217,568 @@ Header            | Description
+response = requests.get(request_url,
+                        auth=(username, password),
+                        params=req_params,
+                        headers=headers,
+                        stream=True)
+
+print response.status_code
+for line in response.iter_lines():
+  # Load json object and print it out
+  json_record = json.loads(line)
+  print json_record
+```
+
+```java
+// This example uses java Unirest library http://unirest.io/java.html
+
+// Request Parameters
+store = "android";      // Could be either "android" or "itunes".
+countryCode = "US";     // Two letter country code.
+date = "{{ date }}";    // Date in YYYY-MM-DD format.
+
+// Auth Parameters
+username = "{API_KEY}"; // Replace {API_KEY} with your own API key.
+password = "X";         // Password can be anything.
+
+// Request URL
+requestUrl = "https://api.appmonsta.com/v1/stores/" + store + "/rankings/aggregate.json";
+
+// Java Main Code Sample
+HttpResponse response = Unirest.get(requestUrl)
+  // This header turns on compression to reduce the bandwidth usage and transfer time.
+  .header("Accept-Encoding", "deflate, gzip")
+  .basicAuth(username, password)
+  .queryString("country", $countryCode),
+  .queryString("date", $date),
+  .asString();
+
+int status = response.getStatus();
+
+BufferedReader in = new BufferedReader(new InputStreamReader(response.getRawBody()));
+String line = null;
+while((line = in.readLine()) != null) {
+  System.out.println(line);
+}
+```
+
+```php
+<?php
+// Request Parameters
+$store = "android";      // Could be either "android" or "itunes".
+$countryCode = "US";     // Two letter country code.
+$date = "{{ date }}";    // Date in YYYY-MM-DD format.
+
+// Auth Parameters
+$username = "{API_KEY}"; // Replace {API_KEY} with your own API key.
+$password = "X";         // Password can be anything.
+
+// Request URL
+$url = "https://api.appmonsta.com/v1/stores/$store/rankings/aggregate.json?country=$countryCode&date=$date";
+
+// PHP Main Code Sample
+$ch = curl_init();
+curl_setopt($ch, CURLOPT_URL,$url);
+curl_setopt($ch, CURLOPT_USERPWD, $username . ":" . $password);
+curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
+curl_setopt($ch, CURLOPT_CONNECTTIMEOUT, 500);
+curl_setopt($ch, CURLOPT_WRITEFUNCTION, function($curl, $data) {
+    // Load json object and print it out
+    $json_record = json_decode((string)$data, true);
+    echo json_encode($json_record);
+    return strlen($data);
+});
+curl_exec($ch);
+curl_close($ch);
+?>
+```
+
+> The above code loads one JSON per line and prints it out:
+
+```json
+{"ranks":["com.free.daily.horoscope.zodiac.secret","com.valpak.android",
+          "com.srpinfosoft.Claptoflashlightonoff","com.tuya.smart",...],
+ "country":"US",
+ "rank_id":"apps_movers_shakers",
+ "genre_id":"LIFESTYLE"}
+
+{"ranks":["479516143","763692274","1396301917","640364616",...],
+ "country":"US",
+ "rank_id":"toppaidapplications",
+ "genre_id":"7002"}
+```
+
+
+Request aggregate rankings by date and by country.
```

Comment:
Add: This call returns records with only ranked app ids per country (not other meta-data). It's ideal if you're only interested in app rank positions.

Reply:
typo: aggregated rankings

Reply:
and yes API call has `aggregate` in the path (which is right - it's used as a noun whereas aggregates is used as a verb).